Sign_in_with_users_Google_Account.feature

```diff
@@ -0,0 +1,94 @@
+Feature: 1.1.1.1 Sign_in_with_users_Google_Account
+Background: 
+    Given user went on 'https://qa.ddso-spot.quantori.com/' page
+    And user clicked on 'Sign In with Google' button
+    And there is a registered account with email 'login111@gmail.com' and phone number '+7nnnnnnnnnn' and password 'password1'
+
+Scenario Outline: 1.1.1.1_01 Signing in with valid email and password
+    Given there are no suggested accounts in the browser
+	When user types 'login111@gmail.com' in the 'Email or phone' field
+	And user clicks the 'Next' button
+	And user types 'password1' in the 'Enter your password' field
+	And user clicks the 'Next' button
+	Then the main page opens
+
+Scenario Outline: 1.1.1.1_02 Signing in with suggested google account
+    Given there is a saved google account in the browser
+	When user clicks on the suggested account button (login111)

```

Comment:
I think examples (login111) should be given as
Examples:
  |login|
  |login111| 
Lecture slides p. 23
Then you will have "When user clicks on <login>"..... 
and examples in the end. I think it will be more general way with specific examples which is nice. Next scenarios also might be apdated accordingly

Reply:
I believe It's only needed when you want to collapse different scenarios in one test case. Here I have only one example so I feel that It might be OK ¯\_(ツ)_/¯